tests/models/test_model.py

```diff
@@ -1930,3 +1933,99 @@ def test_head_dim_8_triton_mqa_attn(batch_size: int = 2):
         output = model(batch)
 
     assert not torch.isnan(output.logits).any()
+
+@pytest.mark.world_size(2)
```

Comment:
Shouldn't this be tested on 4 GPUs minimum?
The TP'd layers need to be fsdp'd in the other dim.